### PR TITLE
[Task] UI tests update

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
## Purpose

* Fixed participants tests
* Updated settings page to handle toggle event for UI tests
* Fixed tests that use settings page

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

### CODE REVIEW

File: eng/pipelines/jobs/templates/set-appcenter-devices.yml [hunk 1]
```diff
@@ -118,7 +118,7 @@ steps:
         Saturday)
           case $HOUR_OF_DAY in
             00)
-              APPCENTER_DEVICES=iphone11promaxset
```
Comment (vhuseinova-msft) [inline eng/pipelines/jobs/templates/set-appcenter-devices.yml:121]: `iphone11promaxset` was run on Saturday and Monday, changed 1 of the runs to `iphone14set` run

File: eng/pipelines/jobs/appcenter-test-job-steps.yml [hunk 2]
```diff
@@ -127,9 +127,9 @@ steps:
       archiveFile: '${{ parameters.archivePath }}/AzureCommunicationUIDemoApp.zip'
 
   - task: UseDotNet@2
-    displayName: "Use .NET Core sdk 3.1.x"
+    displayName: "Use .NET Core sdk 2.1.x"
     inputs:
-      version: 3.1.x
+      version: 2.1.x
```
Comment (jpeng-ms) [inline eng/pipelines/jobs/appcenter-test-job-steps.yml:132]: why did we downgrade the version for .net?

Comment (vhuseinova-msft) [reply eng/pipelines/jobs/appcenter-test-job-steps.yml:132]: It's a temporary fix with a version required by code signing step. Emlyn has created a new PR for updating the code signing step

Comment (jpeng-ms) [reply eng/pipelines/jobs/appcenter-test-job-steps.yml:132]: isn't code sign working before? I am confused by how it's not working now

Comment (emlynmac) [reply eng/pipelines/jobs/appcenter-test-job-steps.yml:132]: This is fixed with the PR here: https://github.com/Azure/communication-ui-library-ios/pull/583

Comment (vhuseinova-msft) [reply eng/pipelines/jobs/appcenter-test-job-steps.yml:132]: Thanks @emlynmac!
@jpeng-ms please see the failed pipeline #20230105.14 for more details

Comment (vhuseinova-msft) [reply eng/pipelines/jobs/appcenter-test-job-steps.yml:132]: Updated with the latest changes to the pipeline

**Review by emlynmac — APPROVED** [review]

**Review by jimchou-dev — APPROVED** [review]

**Pull request merged**